cqrs-user-query-service/src/main/java/dev/amir/userquery/framework/input/rabbitmq/handler/UserMessageHandlerImpl.java

```diff
@@ -15,7 +17,11 @@ public class UserMessageHandlerImpl implements UserMessageHandler {
 
     @Override
     public void handle(SaveUserMessage message) {
+        log.info("Handling incoming message: {}.");
```

Comment:
here you can add the variable message to be printed at the end of the method

`log.info("Handling incoming message: {}.", message);`